test/test_advisory.py

```diff
@@ -158,3 +158,26 @@ def test_reporter_cant_edit_advisory(db, client):
     assert Forbidden.code == resp.status_code
     assert_advisory_data(DEFAULT_ADVISORY_ID, workaround='yay', impact='meh')
     assert 1 == advisory_count()
+
+
+def test_advisory_get_impact_from_text(db, client):
+    impact = advisory_get_impact_from_text(DEFAULT_ADVISORY_CONTENT)
+    assert 'Robots will take over' in impact
+    assert 'Impact' not in impact
+    assert 'References' not in impact
+
+    assert advisory_get_impact_from_text('test') == None
```

Comment:
this tests a completely own thing, this should be moved to a own test case and the method names describing the difference